src/EnergyPlus/OutputProcessor.cc

```diff
@@ -493,6 +493,21 @@ namespace OutputProcessor {
 
 	}
 
+
+	void 
+	SetTimeValueMinutes( Real64 minuteZone, Real64 minuteHVAC )
+	{
+		// set the current minute variables for this array which is an anonymous namespace and so cannot be accessed from other files
+		TimeValue( 1 ).CurMinute = minuteZone;
+		TimeValue( 2 ).CurMinute = minuteHVAC;
+	}
+
+	void 
+	SetOutputProcessorReportNumberCounter( int curReportNumberCounter )
```

Comment:
@JasonGlazer I don't think this function is needed either. You only use it once within the unit test and you set the value to 0. The ReportNumberCounter will be 0 at the beginning of the unit test so you do not need to set the value.